### ISSUE TITLE
using pipe in command results in SIGTERM not being sent

### ISSUE THREAD

**Issue by kilianc** [tj/mon#37]
related to #36

I have something like `"node server.js | rtail --id api-dev"` as mon command and `SIGTERM` never gets to the node process.

**Comment by marcbachmann** [comment]
It would be awesome if this works.
How to reproduce: 
```bash
$ mon "docker run -i -t ubuntu /bin/bash"
$ kill -TERM $PIDofMonProcess
```

**Comment by kilianc** [comment]
I think this may be related to a missing `pipefail` option. I probably need to use `bash -c -o pipefail "node server.js | rtail --id api-dev"` to make it work

**Comment by marcbachmann** [comment]
Yes. In both cases the SIGTERM command isn't handled. Nearly the same problem I have with docker.
